### PR TITLE
Add lots of things again.

### PR DESCRIPTION
**Pull request by Rdzanuwu**

### CODE REVIEW

File: HTML/Melodic Message.html [hunk 1]
```diff
@@ -29,6 +29,9 @@ <h2>On the Subject of Melodic Message</h2>
                     <li>One of keys will only have one letter as a clue. This letter is the one assigned to this key.</li>
                     <li>The clues are shown in a random order.</li>
                     <li>There are six letters in the keyword. The other six letters are not used.</li>
+					<li>Every letter can be used up to 2 times, one as a key, one as a clue.</li>
+					<li>Every key is unique.</li>
+					<li>A key will not have the same letter for both of its clues.</li>
```
Comment (MrPeanut1028) [inline HTML/Melodic Message.html:34]: The spacing should be line up with the `<li>`'s above.

File: HTML/SimpSquad.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <div class="page page-bg-01">
+            <div class="page-header">
+                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                <span class="page-header-section-title">SimpSquad</span>
+            </div>
+            <div class="page-content">
+                <img src="img/Component/SimpSquad.svg" class="diagram">
+                <h2>On the Subject of SimpSquad</h2>
+                <p class="flavour-text">Flavor text fdhusjifkahusdjfkahujdskfiasik</p>
+                </br>
```
Comment (MrPeanut1028) [inline HTML/SimpSquad.html:23]: Preferably, use `<br>` instead of `</br>`.

File: HTML/SimpSquad.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                   With multiple SimpSquads on a bomb, there will be several “chains”, series of blue SimpSquads that are linked to each other. </br></br>
+                   To follow a chain, first press a blue SimpSquad and note down which module it solves. Press the button on that module and observe which module it solves. Continue this until the initially pressed SimpSquad is solved. </br><strong>This will form one chain.</strong></br></br> 
+                   Note that in the event that pressing a blue SimpSquad solves itself, that will still equal a chain of length 1.</br></br>
+                   To solve the red SimpSquad, first solve all other blue SimpSquads and note down the number of chains within the bomb modulo 100. </br></br>
+                   Press the red SimpSquad when the digits of the bomb’s countdown timer contain this number.</br></br></br></br>
+ 
+                   In the event that there is only one SimpSquad on the bomb, there are no chains present, so press the Button when a 0 is present in the countdown timer.</p>
+</br></br><p class="flavour-text">Special thanks to Asew54321, miki2003pl, and Danny7007 for deducing the manual.</p>
```
Comment (MrPeanut1028) [inline HTML/SimpSquad.html:36]: This spacing definitely needs to be cleaned up. In this file as a whole, make sure all opening tags have a closing tag that there's no unnecessary tags.

File: HTML/Tribal Council.html [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        <div class="page page-bg-01">
+            <div class="page-header">
+                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                <span class="page-header-section-title">Tribal Council</span>
+            </div>
+            <div class="page-content">
+                <img src="img/Component/Tribal Council.svg" class="diagram">
+                <h2>On the Subject of Tribal Council</h2>
+                <p class="flavour-text">39 DAYS...SOME AMOUNT OF PEOPLE...1 SURVIVOR!!! I really hope that is not copyrighted…</p>
+                <p>Welcome to Survivor! You are one of the lucky 20 contestants that get to experience the adventure of a lifetime! .</p>
```
Comment (MrPeanut1028) [inline HTML/Tribal Council.html:28]: Punctuation at the end looks off.

File: JSON/SimpSquad.json [hunk 5]
```diff
@@ -0,0 +1,24 @@
+{
+  "Author": "BakersDozenBagels",
```
Comment (MrPeanut1028) [inline JSON/SimpSquad.json:2]: This line needs to be removed if you use the Contributors section.

**Review by MrPeanut1028 — CHANGES_REQUESTED** [review]

File: HTML/Cosmic.html [hunk 6]
```diff
@@ -0,0 +1,39 @@
+﻿<!DOCTYPE html>
+<html>
+    <head>
+        <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+        <title>Cosmic — Keep Talking and Nobody Explodes Module</title>
+        <meta name="viewport" content="initial-scale=1">
```
Comment (samfundev) [inline HTML/Cosmic.html:6]:
```suggestion
        <meta charset="utf-8">
        <meta name="viewport" content="initial-scale=1">
        <title>Cosmic — Keep Talking and Nobody Explodes Module</title>
```

File: HTML/SimpSquad.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <title>SimpSquad — Keep Talking and Nobody Explodes Module</title>
+    <meta name="viewport" content="initial-scale=1">
```
Comment (samfundev) [inline HTML/SimpSquad.html:6]:
```suggestion
    <meta charset="utf-8">
    <meta name="viewport" content="initial-scale=1">
    <title>SimpSquad — Keep Talking and Nobody Explodes Module</title>
```

File: HTML/Melodic Message.html (hunk 1 above)
Comment (samfundev) [inline HTML/Melodic Message.html:34]:
```suggestion
                    <li>Every letter can be used up to 2 times, one as a key, one as a clue.</li>
                    <li>Every key is unique.</li>
                    <li>A key will not have the same letter for both of its clues.</li>
```

**Review by samfundev — CHANGES_REQUESTED** [review]

File: HTML/Cosmic.html (hunk 6 above)
Comment (MrPeanut1028) [reply HTML/Cosmic.html:6]: Are we changing all of the manuals to this format? It looks like the template manual was used here. 
```HTML
<!DOCTYPE html>
<html>
<head>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <title>Module Name — Keep Talking and Nobody Explodes Module</title>
    <meta name="viewport" content="initial-scale=1">
    <link rel="stylesheet" type="text/css" href="css/font.css">
    <link rel="stylesheet" type="text/css" href="css/normalize.css">
    <link rel="stylesheet" type="text/css" href="css/main.css">```

Comment (samfundev) [reply HTML/Cosmic.html:6]: Yes, all manuals should use this new format. That's an old version of the template manual. [The template manual has been updated to use the new format.](https://github.com/Timwi/KtaneContent/blob/master/More/Template%20Manual.zip)

**Review by MrPeanut1028 — APPROVED** [review]

**Pull request merged**